CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 
 tbd
 
+## v1.30.2 - 2024-02-15
+
+- Update base image for tanzu buildpack from cflinuxfs3 to cflinuxfs4.
```

Comment:
Probably should use cloud foundry instead of tanzu. I think tanzu isn't the only user of these.